src/utils/registry.ts

```diff
@@ -203,13 +204,24 @@ export class Registry {
 
   constructor(packagePath: string) {
     const browsersJSON = JSON.parse(fs.readFileSync(path.join(packagePath, 'browsers.json'), 'utf8'));
-    this._descriptors = browsersJSON['browsers'];
+    this._descriptors = browsersJSON['browsers'].map((obj: any) => {
+      const name = obj.name;
+      const revisionOverride = (obj.revisionOverrides || {})[hostPlatform];
+      const revision = revisionOverride || obj.revision;
+      const browserDirectory = revisionOverride ? `${name}-${hostPlatform}-special-${revision}` : `${name}-${revision}`;
```

Comment:
do we really need 'special' in the name, maybe drop it?

Reply:
I want to make sure it stands out. Otherwise, the name `webkit-mac10.14-1234` is not eye-catching in any way, given that I'm on mac10.14